test/common/src/main/kotlin/org/codice/compliance/utils/decorators/IdpResponseDecorators.kt

```diff
@@ -0,0 +1,56 @@
+package org.codice.compliance.utils.decorators
+
+import org.codice.compliance.saml.plugin.IdpPostResponse
+import org.codice.compliance.saml.plugin.IdpRedirectResponse
+import org.w3c.dom.Node
+import javax.xml.parsers.DocumentBuilderFactory
+
+interface IdpResponseDecorator {
```

Comment:
✏️ My recommendation for simplifying this file is in the linked gist. Primarily, there's no apparent use for this interface as you don't need it polymorphically.

https://gist.github.com/coyotesqrl/94119923bea2d3c0573064cdac5830dd